GitCommands/Git/EnvironmentConfiguration.cs

```diff
@@ -60,6 +66,11 @@ public static void SetEnvironmentVariables()
                 Env.SetEnvironmentVariable("SSH_ASKPASS", "ssh-askpass");
             }
 
+            if (!string.IsNullOrEmpty(Env.GetEnvironmentVariable("SSH_ASKPASS")))
```

Comment:
❓ If intentionally, add a comment please. Otherwise
```suggestion
            if (string.IsNullOrEmpty(Env.GetEnvironmentVariable("SSH_ASKPASS")))
```

Reply:
No, it adds `SSH_ASKPASS_REQUIRE` only if `SSH_ASKPASS` is present, since it would be meaningless if it is absent (No program to call). Technically, the only case where this `if` isn't entered is if `File.Exists(sshAskPass)` is false above.

Reply:
I just did not notice the suffix "_REQUIRE".